arangod/RocksDBEngine/RocksDBDumpContext.cpp

```diff
@@ -452,28 +452,34 @@ void RocksDBDumpContext::handleWorkItem(WorkItem item) {
       continue;
     }
 
-    auto storedSlice = [&]() -> VPackSlice {
+    // Sanitize document
+    VPackValueLength const inputLength = documentSlice.byteSize();
+    VPackBuffer<uint8_t> tmpBuffer;
```

Comment:
we should make this a member and call `clear` on every run. To hopefully reuse the memory.

Reply:
I have placed it outside of loop, the same way as `projectionsBuilder`